openhab/openhab-ios
# Alamofire 5

The initiative #551 was stalled. Therefore, I restarted a new initiative with the recommendation to overwrite SessionDelegate 
https://github.com/Alamofire/Alamofire/issues/2886#issuecomment-517951747.

-  Usage of Swift Result Type
-  Migration from SessionManager to Session
-  networkReachabilityStatus changed to status
-  stream to responseStream
-  DataResponse now dually generic in Data, AFError
-  OpenHABAccesTokenAdapter now conformant to RequestInterceptor
-  Defined new OpenHABSessionDelegate conformant to SessionDelegate to handle client certificate handling
-  ServerCertificateManager now conformant to ServerTrustManager and ServerTrustEvaluating

Largely untested:
- Check for stream VideoUITableViewCell
- Check for all types of authentication
- Check for connection to myopenhab.org - Currently requires to set "Always send credentials"

Signed-off-by: Tim Müller-Seydlitz <timbms@gmail.com>
